jsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+	"compilerOptions": {
+		"jsx": "react-jsx",
+		"paths": {
+			// e.g. "@automattic/jetpack-components"
+			// will resolve to "projects/js-packages/components"
+			"@automattic/jetpack-*": [ "./projects/js-packages/*" ]
+		}
+	},
+	// In order to improve performance, especially in VS Code
+	// Let us exclude large/irrelevant files/directories
+	"exclude": [
+		// external packages/modules
+		"**/node_modules/*",
+		"**/vendor/*",
+		// internal directories to exclude
+		"**/build/*",
+		"**/tests/*",
+		"**/wordpress/*",
+		"**/.cache/*",
+		// exclude large (auto-generated) files
+		"**/*.bundle.js",
+		"projects/plugins/jetpack/_inc/blocks/*"
+	]
```

Comment:
`exclude` here is required to fix TS performance issue and this warning every time you open VS Code.

<img width="482" alt="Screenshot 2022-04-05 at 3 50 21 PM" src="https://user-images.githubusercontent.com/18226415/161742266-2311e4f8-ed4f-466e-acb0-4b411d75afa6.png">

Reply:
We could avoid it if we do like #23823 instead.